Modules/HMPID/include/HMPID/HmpidTaskClusters.h

```diff
@@ -0,0 +1,62 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// // distributed under the terms of the GNU General Public License v3 (GPL
+// // Version 3), copied verbatim in the file "COPYING".
+// //
+// // See http://alice-o2.web.cern.ch/license for full licensing information.
+// //
+// // In applying this license CERN does not waive the privileges and immunities
+// // granted to it by virtue of its status as an Intergovernmental Organization
+// // or submit itself to any jurisdiction.
+//
+
+///
+/// \file   HmpidTaskClusters.h
+/// \author Annalisa Mastroserio
```

Comment:
Is it really Annalisa ?

Reply:
Yes, it is really Annalisa, she is helping us for the QC.

Reply:
very good, why is she not creating the PR herself ? any trouble doing it ?